src/distilabel/pipeline/base.py

```diff
@@ -150,15 +150,11 @@ def _create_signature(self) -> str:
 
         steps_info = []
         pipeline_dump = self.dump()["pipeline"]
+
         for step in pipeline_dump["steps"]:
             step_info = step["name"]
             for argument, value in sorted(step["step"].items()):
-                if (
-                    (argument == TYPE_INFO_KEY)
-                    or (argument == "llm")
-                    or (value is None)
-                ):
-                    # NOTE: Should we include the LLM info at this stage??
+                if (argument == TYPE_INFO_KEY) or (value is None):
```

Comment:
I believe there was a rationale for including the `argument == "llm"` there if I'm not mistake, can you confirm @gabrielmbmb

Reply:
I'm not really sure of why those arguments weren't included the first time, but anyway we will have to think of including those when the structured outputs integration is ready as we couldn't take that into account yet.